infra/util/src/main/java/org/apache/shardingsphere/infra/util/expr/InlineExpressionParser.java

```diff
@@ -65,7 +65,15 @@ public static String handlePlaceHolder(final String inlineExpression) {
      * @return result list
      */
     public List<String> splitAndEvaluate() {
-        return Strings.isNullOrEmpty(inlineExpression) ? Collections.emptyList() : flatten(evaluate(split()));
+        if(Strings.isNullOrEmpty(inlineExpression) ){
```

Comment:
Can you add some unit test for this logic?

Reply:
OK